docs/contexts.rst

```diff
@@ -1,50 +0,0 @@
-..
-    Copyright 2017 - Swiss Data Science Center (SDSC)
-    A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
-    Eidgenössische Technische Hochschule Zürich (ETHZ).
-
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-        http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
-
-Contexts and Executions
```

Comment:
Can we think about an API for submitting/listing jobs?

Reply:
yes -- but as above, until it's there I don't really want these extra docs laying around